scripts/map.js

```diff
@@ -597,10 +592,9 @@ var loadDHHL = function () {
         fillPattern: stripes,
       },
     })
-      .bindTooltip(
-        ' Lands owned by the State of Hawaii Department of Hawaiian Homelands as of October, 2022'
-      )
-      .addTo(map)
+    .bindTooltip(
```

Comment:
The indention implies a checkbox will be present? Does that render differently? 
Nvrmnd, gonna test and ask a robot. 
Thanks for pointing that out. 
:fishsticks: